xbmc/interfaces/python/XBPython.cpp

```diff
@@ -581,6 +581,10 @@ bool XBPython::OnScriptInitialized(ILanguageInvoker *invoker)
       CLog::Log(LOGDEBUG, "PYTHONHOME -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
       CLog::Log(LOGDEBUG, "PYTHONPATH -> %s", CSpecialProtocol::TranslatePath("special://frameworks").c_str());
     }
+
+#if defined(TARGET_DARWIN)
```

Comment:
There is OSX specific startup code: SDLMain.mm. Why not putting it there?

> Would a check before setting it in code be acceptable?

Just set the overwrite flag to 0:
setenv("SSL_CERT_FILE", blabla, 0);